src/main/java/org/hashids/Hashids.java

```diff
@@ -10,379 +10,370 @@
  * database IDs, use them as forgotten password hashes, invitation codes, store shard numbers.
  * <p>
  * This is implementation of http://hashids.org v1.0.0 version.
- *
+ * <p>
```

Comment:
This `<p>` has a purpose here, please keep it. If there are some incorrect placements on other comments, it's better to have a specific PR for the change.